testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/interceptor/serverside/ServerInterceptor.java

```diff
@@ -0,0 +1,22 @@
+package org.jboss.as.test.integration.ejb.interceptor.serverside;
+
+import javax.interceptor.AroundInvoke;
+import javax.interceptor.InvocationContext;
+import org.jboss.logging.Logger;
+
+/**
+ * Server side sample interceptor
+ */
+public class ServerInterceptor {
+
+    public ServerInterceptor() {
```

Comment:
We need to make sure that the interceptor was executed. static CountDownLatch possibly?

Reply:
Added, could you please review it? Thanks.